blog/_src/posts/2017-06-05-trip-report-pliss-2017.md

```diff
@@ -0,0 +1,130 @@
+    his [articles][mm-sa] and an older talk he's given. Impressively, the entire
+    example was live-coded, with only a single mistake!
+
+  - **Testing Language Implementations**, by [Alastair Donaldson][ad]. This was
+    an entertaining talk, since Ally showed multiple examples of crashing
+    compilers, and causing other kinds of mischief by triggering compiler bugs.
+
+If you're disappointed that you couldn't see these talks, don't worry! The talks
+were recorded and will be posted very shortly.
+
+## The People
+
+But there's more to PLISS than the talks. Otherwise, I could just watch the
+videos from the comfort of my own home. I'm referring to *networking*, or the
+opportunity to get out and talk to other people about research.
+
+As an early graduate student, I've been given a lot of advice about talking to
+people at conferences and the importance of the "hallway track." I still have
+difficulty doing this at an actual conference, like [PLDI][] or [ECOOP][]. When
+there are hundreds of attendees, or when people already know each other and are
+in conversation groups, I find it difficult to approach them.
+
+This was not the case at PLISS. There were fewer attendees: about fifty students
+and a dozen speakers. There was a good mix of undergraduate, master's,
+first-year PhD, and more senior PhD students. All our breakfasts, lunches, and
+breaks were together, so we would see the same people again and again, and
+inevitably start to learn each other's names. The speakers would also be among
+us, and there was a good ratio of speakers to students for discussions and
+mealtime mentoring.
+
+I had many opportunities to practice my "research pitch." I talked to senior
+students and got advice. I talked to junior students and gave advice. Two
+different people I talked to about my research pointed me to the same paper to
+read. I found another student who was working with [IFDS][], an algorithm I have
+spent much time trying to understand. And, one day at lunch, my table discovered
+that we were all working on static analysis!
+
+As much as I enjoyed the talks, I think the best part of PLISS was meeting and
+talking to other people. You can replace talks with videos (but you lose the
+speaker-audience interaction), and you can replace conversations with other
+forms of communication. But there isn't really anything that can replace the
+serendipity of bumping into someone with a shared interest.
+
+## The Location
+
+Actually, the *other* best part of PLISS was the location. Italy is a beautiful
+country with delicious food. And Bertinoro is a small town on the top of a hill,
+with a breathtaking view of the surrounding area. The lectures were held in
+a [castle at the top of the hill][castle] (photo credit: Steve Blackburn). The
+speakers lived in the castle for the week, while the students lived in the
+former monastery, which has been renovated into a university residence.
```

Comment:
there is really an opportunity for a joke/jab here: `former monastery (seems fitting)`. But maybe you intended to be lightweight about the joke?